.eslintrc

```diff
@@ -24,11 +24,16 @@
     "max-len": "off",
     "max-params": ["error", 5],
     "multiline-ternary": "off",
+    "no-eval": "error",
+    "no-implied-eval": "error",
+    "no-new-func": "error",
     "no-param-reassign": ["error", { "props": false }],
     "no-plusplus": ["error", { "allowForLoopAfterthoughts": true }],
     "no-restricted-syntax": "off",
     "no-trailing-spaces": "off",
     "no-underscore-dangle": "off",
+    "no-unsanitized/method": "error",
```

Comment:
What do you think of using the ` "extends": ["plugin:no-unsanitized/DOM"]` syntax instead so we can reduce the number of custom rules that we have? I'd like to work towards using a community config.

Reply:
Yeah can do, I'll get that changed today 👍🏻